### PR TITLE
Run bundler test spec

### PR DESCRIPTION
**Pull request by zsluedem**

### CODE REVIEW

File: .github/workflows/bundle_spec_test.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+on:
+  workflow_run:
+    workflows: ["Create and publish a Docker image"]
+    types: [completed]
+
+name: run bundler test spec
```
Comment (Vid201) [inline .github/workflows/bundle_spec_test.yml:6]: Change to "Run bundler spec tests"

File: .github/workflows/bundle_spec_test.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      - run: cd bundler-spec-tests; git checkout
+        name: "re-checkout bundler-spec-tests (on top of cache)"
+
+        #for faster "update-deps" for spec
+      - run: yarn --cwd bundler-spec-tests/spec remove gatsby || echo "already removed"
+
+      - run: "cd bundler-spec-tests && pdm install && pdm update-deps"
+        name: update submodules of bundler-spec-tests
+
+      - run: ./runall.sh launchers/aabundler-launcher.sh
```
Comment (Vid201) [inline .github/workflows/bundle_spec_test.yml:42]: Shouldn't this be `./runall.sh launchers/aa-bundler-rust-launcher.sh` ?

**Review by Vid201 — CHANGES_REQUESTED** [review]
Looks good, makes sense to build docker image only on new tags!

File: .github/workflows/bundle_spec_test.yml (hunk 2 above)
Comment (zsluedem) [reply .github/workflows/bundle_spec_test.yml:42]: You are right

**Review by Vid201 — APPROVED** [review]

**Pull request merged**